src/coreneuron/io/reports/nrnreport.hpp

```diff
@@ -90,7 +90,7 @@ struct ReportConfiguration {
     std::vector<std::string> var_names;   // variable names
     std::vector<int> mech_ids;            // mechanisms
     std::string unit;                     // unit of the report
-    std::string format;                   // format of the report (Bin, hdf5, SONATA)
+    std::string format;                   // format of the report (hdf5, SONATA)
```

Comment:
```suggestion
    std::string format;                   // format of the report (SONATA)
```

we are removing legacy "hdf" format as well